job.go

```diff
@@ -110,13 +101,21 @@ func (j *Job) periodDuration() (time.Duration, error) {
 	return periodDuration, nil
 }
 
-// NextScheduledTime returns the time of when this job is to run next
+// NextScheduledTime returns the time of the Job's next scheduled run
 func (j *Job) NextScheduledTime() time.Time {
 	return j.nextRun
 }
 
-// GetWeekday returns which day of the week the job will run on
-// This should only be used when .Weekday(...) was called on the job.
-func (j *Job) GetWeekday() time.Weekday {
-	return j.startDay
+// GetAt returns the specific time of day the Job will run at
+func (j *Job) GetAt() string {
```

Comment:
Would returning `time.Time` instead of `string` be better here?

Reply:
I agree, time would be better. I'm going to leave that for #14